test/testcafe/spec/EnrollAuthenticatorWebAuthn_spec.js

```diff
@@ -31,12 +36,13 @@ test('should have webauthn not supported error if browser doesnt support', async
   const enrollWebauthnPage = await setup(t);
   await t.expect(enrollWebauthnPage.getFormTitle()).eql('Set up security key or biometric authenticator');
   await t.expect(enrollWebauthnPage.hasEnrollInstruction()).eql(false);
-  await t.expect(enrollWebauthnPage.hasWebauthnNotSupportedError()).eql(true);
+  await t.expect(enrollWebauthnPage.getWebauthnNotSupportedError())
+    .eql('Security key or biometric authenticator is not supported on this browser. Contact your admin for assistance.');
```

Comment:
<img width="422" alt="Screen Shot 2022-12-06 at 9 43 28 AM" src="https://user-images.githubusercontent.com/27521424/206057216-6d085bc0-b10c-407b-9a64-696151de0995.png">
In v2, the unsupported browser is shown as an error.
<img width="457" alt="Screen Shot 2022-12-06 at 4 27 41 PM" src="https://user-images.githubusercontent.com/27521424/206057258-db9f4578-e290-4796-9971-3d8adff8d1a6.png">
In v3, it's shown as a message with `[data-se="o-form-explain"]` 

Shouldn't we show this as an error message in v3?

Reply:
Oh yes, I believe this should be an error in v3 as well. This should be a simple fix, did you want to create a bug for this or did you want me to apply the fix in this branch?

Reply:
Created the bug, we will get this fixed after this is merged.

https://oktainc.atlassian.net/browse/OKTA-556759